hack/e2e-suite/services.sh

```diff
@@ -278,20 +278,28 @@ function wait_for_service_down() {
 #   $5: pod IDs (sorted)
 function verify_from_container() {
   echo "waiting for $1 at $2:$3"
-  results=($(ssh-to-node "${test_node}" "
-      set -e;
-      sudo docker pull gcr.io/google_containers/busybox >/dev/null;
-      sudo docker run gcr.io/google_containers/busybox sh -c '
-          for i in $(seq -s' ' 1 $(($4*3))); do
-            if wget -q -T 3 -O - http://$2:$3; then
-              echo
-            else
-              exit 1
-            fi
-          done
-      '" | sort -r -n | uniq)) \
-      || error "testing $1 VIP from container failed"
-  found_pods=$(sort_args "${results[@]}")
+  # TODO: Reduce this interval once we have a sense for the latency distribution.
```

Comment:
How are we going to get a sense for latency if we don't add some logging here with the time taken?